packages/auth-magic-link-nest/src/magic-link/export.types.ts

```diff
@@ -25,7 +29,7 @@ export interface AuthMagicLinkUtil<DBUser = any> {
     refreshTokenHash: string,
   ) => void | Promise<void>;
   sendMagicLink: (
-    user: DBUser,
+    user: MagicLinkUser,
```

Comment:
basically we have the extracted query params from the API call here which we stated in `magic-link-strategy.ts` in the `userFields` array. I think we should rename the parameter in this function to something like `extractedQueryParams`. Also the interface `MagicLinkUser` should be renamed to something similar.